### PR TITLE
working v1 site builder redesign

### PR DESCRIPTION
**Pull request by abusel**
#### Linear 🎫 
https://linear.app/flok/issue/FLO-508

##### Description
V1 of site builder redesign

### CODE REVIEW

File: modules/flok/src/components/attendee-site/block/AccordionBlock.tsx [hunk 1]
```diff
@@ -175,7 +175,8 @@ export function AccordionBlockEditor(props: AccordionBlockEditorProps) {
 }
 
 type AccordionBlockRendererProps = {
-  block: AttendeeLandingWebsiteBlockModel<AccordionBlockContentModel>
+  // block: AttendeeLandingWebsiteBlockModel<AccordionBlockContentModel>
```
Comment (jaredhanson11) [inline modules/flok/src/components/attendee-site/block/AccordionBlock.tsx:178]: remove comment?

File: modules/flok/src/components/attendee-site/website/SiteHeader.tsx [hunk 2]
```diff
@@ -62,7 +62,12 @@ let useStyles = makeStyles((theme) => ({
   },
 }))
 
-type PageNavItem = {to: string; title: string; active: boolean}
+type PageNavItem = {
+  to: string
+  title: string
+  active: boolean
+  builderTo: string
```
Comment (jaredhanson11) [inline modules/flok/src/components/attendee-site/website/SiteHeader.tsx:69]: don't like this IMO

File: modules/flok/src/redesign/pages/SiteBuilderPage.tsx [hunk 3]
```diff
@@ -0,0 +1,1143 @@
...
+import {PageBlock} from "../site-builder/PageBlock"
+import PageBuilderSideNavBlock from "../site-builder/PageBuilderSideNavBlock"
+import UploadImageWithTemplate from "../site-builder/UploadImageWithTemplate"
+
+const OverallPage = styled("div", {
+  display: "flex",
+  flexDirection: "row",
+  width: "100%",
+  height: "100%",
+  backgroundColor: "#FFF",
```
Comment (jaredhanson11) [inline modules/flok/src/redesign/pages/SiteBuilderPage.tsx:90]: theme

File: modules/flok/src/redesign/pages/SiteBuilderPage.tsx [hunk 4]
```diff
@@ -0,0 +1,1143 @@
...
+  width: "100%",
+  borderBottomStyle: "solid",
+  borderBottomWidth: "1px",
+  borderBottomColor: theme.colors.gray6,
+  display: "flex",
+  flexDirection: "row",
+  padding: "12px 20px",
+  alignItems: "center",
+  justifyContent: "space-between",
+  position: "sticky",
```
Comment (jaredhanson11) [inline modules/flok/src/redesign/pages/SiteBuilderPage.tsx:137]: it's not currently sticky so is this needed? seems like unused/extra css not working as intended anyway so just remove

File: modules/flok/src/redesign/pages/SiteBuilderPage.tsx [hunk 5]
```diff
@@ -0,0 +1,1143 @@
...
+const PortalHeaderButton = styled("div", {
+  height: "10px",
+  width: "10px",
+  borderRadius: "60px",
+  borderStyle: "solid",
+  borderWidth: "0.5px",
+  variants: {
+    color: {
+      red: {
+        background: "#ED6A5E",
```
Comment (jaredhanson11) [inline modules/flok/src/redesign/pages/SiteBuilderPage.tsx:163]: do we have theme for this or these are one offs?

File: modules/flok/src/redesign/pages/SiteBuilderPage.tsx [hunk 6]
```diff
@@ -0,0 +1,1143 @@
...
+  let {openSidebar, closeSidebar, sidebarOpen} = useSidebar()
+  useEffect(() => {
+    closeSidebar()
+  }, [])
+  const isSmallScreen = useMediaQuery((theme: FlokTheme) =>
+    theme.breakpoints.down("sm")
+  )
+
+  const deletablePage = page?.title.toLowerCase() !== "home"
+  let pageNameFormik = useFormik({
```
Comment (jaredhanson11) [inline modules/flok/src/redesign/pages/SiteBuilderPage.tsx:343]: Once we investigate performance issues, we should look if all the formik objects are causing it. In which case we could probably combine multiple formik objects and only patch the changes if the values are different from initial values

File: modules/flok/src/redesign/site-builder/blocks/ImageBlock.tsx [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+              {getEditorImages(props.formik.values[props.block.id]?.content)}
+            </div>
+          </div>
+        </Box>
+      </form>
+    </div>
+  )
+}
+
+let useRendererStyles = makeStyles((theme) => ({
```
Comment (jaredhanson11) [inline modules/flok/src/redesign/site-builder/blocks/ImageBlock.tsx:172]: Remove mui

File: modules/flok/src/redesign/pages/SiteBuilderPage.tsx [hunk 8]
```diff
@@ -0,0 +1,1143 @@
...
+            <SitePortal>
+              <PortalHeader>
+                <PortalHeaderButtons>
+                  <PortalHeaderButton color="red" />
+                  <PortalHeaderButton color="yellow" />
+                  <PortalHeaderButton color="green" />
+                </PortalHeaderButtons>
+                <PortalHeaderUrl>
+                  <Text>
+                    {`app.goflok.com/sites/${titleToNavigation(
```
Comment (jaredhanson11) [inline modules/flok/src/redesign/pages/SiteBuilderPage.tsx:1088]: Should never hard code this in

**Review by jaredhanson11 — APPROVED** [review]

File: modules/flok/src/redesign/pages/SiteBuilderPage.tsx (hunk 3 above)
Comment (abusel) [reply modules/flok/src/redesign/pages/SiteBuilderPage.tsx:90]: lets talk about this

**Pull request merged**